app/views/administrate/application/_collection.html.erb

```diff
@@ -36,12 +36,9 @@ to display a collection of resources in an HTML table.
 
             <% if collection_presenter.ordered_by?(attr_name) %>
               <span class="cell-label__sort-indicator cell-label__sort-indicator--<%= collection_presenter.ordered_html_class(attr_name) %>">
-                <%= svg_tag(
-                  "administrate/sort_arrow.svg",
-                  "sort_arrow",
-                  width: "13",
-                  height: "13"
-                ) %>
+                <svg class="search__clear-icon" aria-hidden="true">
+                  <use xlink:href="#icon-up-caret" />
```

Comment:
This SVG is presentational and only a visual cue, so I’ve added `aria-hidden="true"`, which removes it from the accessibility tree. No `title` or `desc` needed.